### PR TITLE
Update privateca API v1beta1 to v1 

### PR DESCRIPTION
**Pull request by ihtigum**
Updated the cas-gclb-rotation tool from from v1beta1 to v1
- Added v1beta1 to v1
- Updated config.yaml  from issueCa -> issuingPool 
- Change public key format to PEM 
- Added x509 in cert body

### CODE REVIEW

**Comment by mugithi** [comment]
Updated the cas-gclb-rotation tool from from v1beta1 to v1

Added v1beta1 to v1
Updated config.yaml from issueCa -> issuingPool
Change public key format to PEM
Added x509 in cert body

File: app/config.py [hunk 1]
```diff
@@ -38,7 +38,8 @@ def isGlobal(self) -> bool:
 class RotationProfile:
     """Describes the entity to be rotated."""
     lb: SimpleResource
-    issuingCa: SimpleResource
+    ##TODO issueCaPool
```
Comment (pmansour) [inline app/config.py:41]: nit: can remove this line

File: app/config.yaml [hunk 2]
```diff
@@ -13,25 +13,25 @@
 # limitations under the License.
 
 profiles:
-- dnsName: example.com
+- dnsName: lb-cert.isaack.io
```
Comment (pmansour) [inline app/config.yaml:16]: are you sure you want to leave your name in here?

(and same below)

File: app/gcp_clients.py [hunk 3]
```diff
@@ -174,39 +174,45 @@ def __init__(self, discovery_client: discovery.Resource,
...
                 },
-                'reusableConfig': {
-                    'reusableConfig':
-                    'projects/privateca-data/locations/{}/reusableConfigs/leaf-server-tls'
-                    .format(self.location),
+                'x509Config': {
+                    'caOptions': {
+                        'isCa': 'false'
+                        },
+                    'keyUsage': {
```
Comment (pmansour) [inline app/gcp_clients.py:202]: could you explicitly include the key usages and EKUs from here: https://cloud.devsite.corp.google.com/certificate-authority-service/docs/certificate-profile#end-server-tls

specifically:
```
keyUsage:
  baseKeyUsage:
    digitalSignature: true
    keyEncipherment: true
  extendedKeyUsage:
    serverAuth: true
```

**Review by pmansour — APPROVED** [review]
LGTM assuming you've tested this locally, and with the comments mentioned below

**Comment by pmansour** [comment]
Author is OOO, so will merge these changes and follow-up on my comments separately.

**Pull request merged**